stripe/src/test/java/com/stripe/android/StripeTest.java

```diff
@@ -759,6 +759,45 @@ public void createSourceSynchronous_withIDealParamsNoStatement_doesNotIgnoreBank
         }
     }
 
+
+    @Test
+    public void createSourceSynchronous_withIDealParamsNoName_passesIntegrationTest() {
```

Comment:
super nit: I would have kept the capitalization of `iDEAL` consistent between `SourceParams.createIdealParams` and the name of the test - but I'm just now noticing that the test above this also uses `IDeal`, so 🤷‍♂️